tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/wit.py

```diff
@@ -119,6 +121,22 @@ def _delete_example(self, change):
         i if i < index else i - 1 for i in self.updated_example_indices])
     self._generate_sprite()
 
+  @observe('compute_custom_distance')
+  def _compute_custom_distance(self, change):
+    info = self.compute_custom_distance
+    index = info['index']
+    params = info['params']
+    callback_fn = info['callback']
+    try:
+      distances = base.WitWidgetBase.compute_custom_distance_impl(self, index,
+                                                       params['distanceParams'])
+      self.custom_distance_dict = {'distances':distances,
+                                   'exInd':index,
+                                   'callback_fn':callback_fn,
+                                   'params':params['callbackParams']}
+    except Exception as e:
```

Comment:
jupyter has error catching and colab version doesn't. do we have this issue with other methods?

Reply:
added error catching to colab as well.